chain-signatures/node/src/protocol/cryptography.rs

```diff
@@ -136,6 +113,7 @@ impl CryptographicProtocol for GeneratingState {
                         public_key = hex::encode(r.public_key.to_bytes()),
                         "generating: successfully completed key generation"
                     );
+                    // TODO: handle secret storage error
```

Comment:
What kind of handling you want to add?

Reply:
if it doesn't get stored to secret storage, we should store the key in memory and keep retrying